axelor-stock/src/main/java/com/axelor/apps/stock/service/StockMoveLineServiceImpl.java

```diff
@@ -1200,7 +1200,7 @@ public void setAvailableStatus(StockMoveLine stockMoveLine) {
             + stockMoveLine.getProduct().getId()
             + " AND self.id in (select stockLocationLine.trackingNumber.id from StockLocationLine stockLocationLine join StockLocation sl on sl.id = stockLocationLine.detailsStockLocation.id WHERE sl.id = "
```

Comment:
Since you changed this method, please also improve its readability:
- The first line of the method is not necessary, we can declare `trackingNumbers` variable as we set it.
- Split this line so we do not have a line longer than 100 characters (to follow our coding convention)